### ISSUE TITLE
hestiaMEDIA.ToHTML rendered a blocked inline CSS Style

### ISSUE THREAD

**Issue by hollowaykeanho** [ZORALab/Hestia#388]
# Description
```
Please provide a short description of what you have encountered below.
````
`hestiaMEDIA.ToHTML` failed to render inline styling due to Hugo safety protection. It was caused by missing `safeCSS` function. See below screenshot:

![inline-css-rejected-media](https://github.com/ZORALab/Hestia/assets/7259572/1a535e4a-e8ea-4736-86f3-018d4c8e03fe)

# Expected Behavior
```
Please specify the expected behavior of your usage below.
```

Fixed patch:

```
diff --git a/sites/themes/hestiaHUGO/layouts/partials/hestiaMEDIA/ToHTML b/sites/themes/hestiaHUGO/layouts/partials/hestiaMEDIA/ToHTML
index 62d2f53..0c4b3fb 100644
--- a/sites/themes/hestiaHUGO/layouts/partials/hestiaMEDIA/ToHTML
+++ b/sites/themes/hestiaHUGO/layouts/partials/hestiaMEDIA/ToHTML
@@ -61,7 +61,7 @@ specific language governing permissions and limitations under the License.
 {{- if $isVideo -}}
 <video {{- if $media.ID }} id='{{- $media.ID -}}' {{- end }}
        {{ if $media.Class -}} class='{{- $media.Class -}}' {{- end }}
-       {{ if $media.Style -}} style='{{- $media.Style -}}' {{- end }}
+       {{ if $media.Style -}} style='{{- safeCSS $media.Style -}}' {{- end }}
        {{ if $media.Decorative -}} aria-hidden="true" {{- end }}
        {{ if $media.Autoplay -}} autoplay {{- end }}
        {{ if $media.Control -}} controls {{- end }}
@@ -90,7 +90,7 @@ specific language governing permissions and limitations under the License.
 {{- else if $isImage -}}
 <picture {{- if $media.ID }} id='{{- $media.ID -}}' {{- end }}
        {{ if $media.Class -}} class='{{- $media.Class -}}' {{- end }}
-       {{ if $media.Style -}} style='{{- $media.Style -}}' {{- end }}
+       {{ if $media.Style -}} style='{{- safeCSS $media.Style -}}' {{- end }}
        {{ if $media.Decorative -}} aria-hidden="true" {{- end }} />
 
 
@@ -123,7 +123,7 @@ specific language governing permissions and limitations under the License.
 {{- else if $isAudio -}}
 <audio {{- if $media.ID }} id='{{- $media.ID -}}' {{- end }}
        {{ if $media.Class -}} class='{{- $media.Class -}}' {{- end }}
-       {{ if $media.Style -}} style='{{- $media.Style -}}' {{- end }}
+       {{ if $media.Style -}} style='{{- safeCSS $media.Style -}}' {{- end }}
        {{ if $media.Decorative -}} aria-hidden="true" {{- end }}
        {{ if $media.Autoplay -}} autoplay {{- end }}
        {{ if $media.Control -}} controls {{- end }}
```

# Current Behavior
```
Please specify the current behavior of software below.
```
blocked by Hugo's CSS safe rendering.

# Steps to Reproduce (COMPULSORY)
```
Please specify all the steps required to re-produce the problem.
```

1. On a page, render the following codes:
```
{{- $ret = merge $Page (dict "Input" (dict "Data" [A MEDIA DATA])) -}}
{{- $ret = partial "hestiaMEDIA/Sanitize" $ret -}}
{{- $ret = merge $ret.Output (dict "Style" "margin: 0 auto;") -}}
{{ partial "hestiaMEDIA/ToHTML" $ret }}
```
3. Inspect the rendered picture.

# Attachments
```
Please drag and drop the necessary data files (e.g. screenshot, logs, etc)
below.
```

**Comment by hollowaykeanho** [comment]
Patch applied in 4a39825cb9ad452a7d144e11e28c7b626ddbb89d.

**Comment by hollowaykeanho** [comment]
Released in v1.2.1.
